jobsboard/jobs/models.py

```diff
@@ -26,3 +27,6 @@ class Job(TimeStampedModel):
     url = models.URLField()
     is_featured = models.BooleanField(default=False)
     employment_type = models.IntegerField(choices=EMPLOYMENT_TYPE, default=1)
+    location = models.CharField(max_length=256, blank=True)
+    email = models.CharField(max_length=100, blank=True)
```

Comment:
Maybe use `models.EmailField` so we can get free validation when using a FormView?